docs/pbc/all/accessibility-improvement/202410.0/b2b/install-the-spryker-accessibility-improvements.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Install the Spryker core keyboard accessibility improvements
+description: Learn how to the Spryker core keyboard accessibility improvements.
+last_updated: March 31, 2025
+template: feature-integration-guide-template
+---
+
+Keyboard accessibility improvements include the following changes:
+
+- Adjusted multiple components to be more accessible via keyboard:
+  - Added `aria-label` attributes to the multiple components to improve screen reader support
+  - Adjusted `tabIndex` attributes to improve keyboard navigation
+- Add a new attribute `parent-class-name` for the `suggest-search` component that helps to improve keyboard accessibility for the product searching process.
+- Improved color schema for better text to background contrast
```

Comment:
- Enhanced Accessibility: Made several interface elements more accessible. You can now navigate more logically using the Tab key, and screen readers provide better descriptions of elements.

- Keyboard-Friendly Search: Improved the way keyboard users can interact with the product search suggestions.

- Better Visual Clarity: Adjusted the color scheme across the site to ensure text stands out better from its background, making it easier for everyone to read.